qiskit_experiments/library/characterization/fine_amplitude.py

```diff
@@ -99,43 +98,75 @@ def _default_experiment_options(cls) -> Options:
                 by doing :code:`options.gate_class()`.
             normalization (bool): If set to True the DataProcessor will normalized the
                 measured signal to the interval [0, 1]. Defaults to True.
-            add_sx (bool): If True then the circuits will start with an sx gate. This is typically
-                needed when calibrating pulses with a target rotation angle of :math:`\pi`. The
-                default value is False.
-            add_xp_circuit (bool): If set to True then a circuit with only an X gate will also be
-                run. This allows the analysis class to determine the correct sign for the amplitude.
+            add_cal_circuits (bool): If set to True then two circuits to calibrate 0 and 1 points
+                will be added. These circuits are often needed to properly calibrate the amplitude
+                of the ping-pong oscillation that encodes the errors.
         """
         options = super()._default_experiment_options()
-        options.repetitions = list(range(15))
+        options.repetitions = list(range(1, 15))
         options.gate = None
         options.normalization = True
-        options.add_sx = False
-        options.add_xp_circuit = True
+        options.add_cal_circuits = True
 
         return options
 
-    def __init__(self, qubit: int, gate: Gate, backend: Optional[Backend] = None):
+    def __init__(self, qubits: Sequence[int], gate: Gate, backend: Optional[Backend] = None):
         """Setup a fine amplitude experiment on the given qubit.
 
         Args:
-            qubit: The qubit on which to run the fine amplitude calibration experiment.
+            qubits: The qubit(s) on which to run the fine amplitude calibration experiment.
             gate: The gate that will be repeated.
             backend: Optional, the backend to run the experiment on.
         """
-        super().__init__([qubit], analysis=FineAmplitudeAnalysis(), backend=backend)
+        super().__init__(qubits, analysis=FineAmplitudeAnalysis(), backend=backend)
         self.set_experiment_options(gate=gate)
 
-    def _pre_circuit(self) -> QuantumCircuit:
+    def _cal_circuits(self) -> List[QuantumCircuit]:
+        """This method returns the calibration circuits.
+
+        Calibration circuits allow the experiment to overcome state preparation and
+        measurement errors which cause ideal probabilities to be below 1.
+        """
+        cal_circuits = []
+
+        for add_x in [0, 1]:
+            circ = QuantumCircuit(self.num_qubits, self.num_qubits)
+
+            if add_x:
+                circ.x(range(self.num_qubits))
+
+            circ.measure(range(self.num_qubits), range(self.num_qubits))
```

Comment:
This doesn't cause marginalization problem in 2Q module? As far as I know fine amp only measures target qubit where the rotation is induced, I would give measured qubit index rather than measure full qubits and marginalize it later.

Reply:
Good point. This should fix it: [5f39993](https://github.com/Qiskit/qiskit-experiments/pull/655/commits/5f39993f38b4a0d40c4b17b2e68b3f78cb4b51f7)